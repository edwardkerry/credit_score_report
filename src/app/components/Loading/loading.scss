.report__loading {
  width: 100px;
  height: 100px;
  background-size: cover;
  -webkit-animation: fadein 2s ease-in alternate infinite;
  -moz-animation: fadein 2s ease-in alternate infinite;
  animation: fadein 2s ease-in alternate infinite;
  background-image: url('/clearscore.png');
}

@-webkit-keyframes fadein {
  from { opacity: 0.2; }
  to { opacity: 0.8; }
}

@-moz-keyframes fadein {
  from { opacity: 0.2; }
  to { opacity: 0.8; }
}

@keyframes fadein {
  from { opacity: 0.2; }
  to { opacity: 0.8; }
}