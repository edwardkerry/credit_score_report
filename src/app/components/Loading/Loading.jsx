import React from 'react';

import bemHelper from '../../utils/bem';

import './loading.scss';

const cn = bemHelper({ block: 'content' });

export default() => (
  <div className={cn('loading', null)} />
)