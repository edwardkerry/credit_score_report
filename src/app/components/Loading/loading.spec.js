import React from 'react';
import renderer from 'react-test-renderer';
import Loading from './Loading.jsx'

describe('Loading', () => {
  it('renders', () => {
    const component = renderer.create(<Loading />);
    const json = component.toJSON();
    expect(json).toMatchSnapshot();
  });
})