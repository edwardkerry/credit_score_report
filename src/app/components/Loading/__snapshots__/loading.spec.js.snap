// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading renders 1`] = `
<div
  className="report__loading"
/>
`;
